akka-actor-typed/src/main/scala/akka/actor/typed/javadsl/Routers.scala

```diff
@@ -64,6 +64,18 @@ abstract class GroupRouter[T] extends DeferredBehavior[T] {
    */
   def withRoundRobinRouting(): GroupRouter[T]
 
+  /**
+   * Route messages by using consistent hashing.
+   *
+   * From wikipedia: Consistent hashing is based on mapping each object to a point on a circle
+   * (or equivalently, mapping each object to a real angle). The system maps each available machine
+   * (or other storage bucket) to many pseudo-randomly distributed points on the same circle.
```

Comment:
For the factory method with `virtualNodesFactor` we need some elaboration here, the number must be 1 or more, what is a good number and how to you find it, what is the consequence of choosing a too large number.